packages/test-utils/src/wrapper.js

```diff
@@ -711,7 +711,17 @@ export default class Wrapper implements BaseWrapper {
 
       // $FlowIgnore : Problem with possibly null this.vm
       this.vm.$forceUpdate()
-      return nextTick()
+      return new Promise((resolve, reject) => {
```

Comment:
Since nothing is rejected?
```suggestion
      return new Promise(resolve => {
```